.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,8 +16,10 @@
 * **What is the new behavior?**
 <!-- if this is a feature change -->
 
-- **Does this PR introduce a breaking change?**
-<!-- What changes might users need to make in their workflow or application due to this PR? -->
+- **Does this PR introduce a breaking change, and is
+[titled accordingly](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#breaking)?**
+<!-- If this PR is merged, will any users need to change their code, command-line invocations, build scripts or data files
+when upgrading from an older version of arduino-cli? -->
```

Comment:
```suggestion
when upgrading from an older version of Arduino CLI? -->
```
My understanding is that the name of the application is "Arduino CLI" and `arduino-cli` is only the command name. Similar to Git vs. `git`.